.github/workflows/update-gaffer-version.yaml

```diff
@@ -14,32 +14,40 @@ jobs:
       VERSION_UPDATE_BRANCH: updating-gaffer-version-${{ github.event.inputs.version }}
 
     steps:
-    - name: Checkout branch
-      uses: actions/checkout@v2
-      with:
-        token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
-
-    - name: Get branch name
-      run: echo "SOURCE_BRANCH=$(git branch --show-current)" >> $GITHUB_ENV
-
-    - name: Set up Github credentials
-      run: |
-        git config user.name Gaffer
-        git config user.email github-actions@github.com
+    - name: Checkout base branch
+      uses: actions/checkout@v3
 
     - name: Setup JDK
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v3
       with:
         distribution: 'zulu'
         java-version: '8'
 
     - name: Update Gaffer Version
-      run: ./cd/updateGafferVersion.sh ${{ github.event.inputs.version }}
+      run: |
+        mvn -q org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.parent.version
+        oldVersion=`mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.parent.version | grep -v '\['`
+        newVersion=${{ github.event.inputs.version }}
+
+        sed -i'' "s#version>$oldVersion<#version>$newVersion<#g" pom.xml
+        sed -i'' "s#gaffer2:$oldVersion#gaffer2:$newVersion#g" NOTICES
+
+        sed -i'' "s#__version__ = \"$oldVersion\"#__version__ = \"$newVersion\"#g" python-shell/__init__.py
+        sed -i'' "s#__version__ = \"$oldVersion\"#__version__ = \"$newVersion\"#g" python-shell/src/__init__.py
+        sed -i'' "s#__version__ = \"$oldVersion\"#__version__ = \"$newVersion\"#g" python-shell/src/*/__init__.py
+
+    - name: Update gafferpy
+      run: |
+        newVersion=${{ github.event.inputs.version }}
+        curl -o spring-rest.jar https://repo.maven.apache.org/maven2/uk/gov/gchq/gaffer/spring-rest/$newVersion/spring-rest-$newVersion-exec.jar
+        java -Dgaffer.schemas=ui/example/road-traffic/schema/ -Dgaffer.storeProperties=ui/example/federated/federatedStore.properties -Dgaffer.graph.config=ui/example/federated/graphConfig.json -jar spring-rest.jar &
+        sleep 1m
+        python python-shell/src/generate.py
+        rm spring-rest.jar
```

Comment:
I wouldn't bother removing it.

Reply:
It's either that or it gets put into the PR by the action